src/t8_forest/t8_forest_ghost_interface_wrapper.h

```diff
@@ -0,0 +1,80 @@
+/*
+  This file is part of t8code.
+  t8code is a C library to manage a collection (a forest) of multiple
+  connected adaptive space-trees of general element classes in parallel.
+
+  Copyright (C) 2015 the developers
+
+  t8code is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2 of the License, or
+  (at your option) any later version.
+
+  t8code is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with t8code; if not, write to the Free Software Foundation, Inc.,
+  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+*/
+
+#ifndef T8_FOREST_GHOST_INTERFACE_WRAPPER_H
+#define T8_FOREST_GHOST_INTERFACE_WRAPPER_H
+
+#include <t8.h>
+#include <t8_forest/t8_forest_general.h>
+#include <t8_forest/t8_forest_ghost_interface.h>
+
+T8_EXTERN_C_BEGIN ();
+
+/**
+ * Satisfy the C interface of forest
+ * Create a new ghost_interface of type face with given version
+ */
+t8_forest_ghost_interface_c *
+t8_forest_ghost_interface_face_new (int version);
+
+/**
+ * Satisfy the C interface of forest
+ * Create a new ghost_interface of type stencil
+ */
+t8_forest_ghost_interface_c *
+t8_forest_ghost_interface_stencil_new ();
+
+/**
+ * Satisfy the C interface of forest
+ * Return for a ghost_interface of Type FACE the ghost_algorithm / ghost_version (1, 2 or 3)
```

Comment:
Please add a disclaimer that this function only works for ghost objects of the face class

Reply:
In the discription of the parameter, stands that is must be this class.
`\param [in]    ghost_interface Pointer to object of class t8_forest_ghost_face`

Is there a possibility to make this more clear? I could use `\note`  or `\remark` or something else?

Reply:
Yes you can use \note